app/HieWrapper.hs

```diff
@@ -90,9 +90,9 @@ run opts = do
       args <- getArgs
       logm $ "args:" ++ show args
       logm "launching ....\n\n\n"
-      callProcess e args
+      callProcess e ("--lsp" : args)
```

Comment:
Not sure if we should do this by default, there does exist a non-lsp JSON mode that should be able to use the wrapper too

Reply:
This was mostly because of the way both the vscode and atom plugins handled custom wrappers. It could probably be removed.

Reply:
Looks like https://github.com/alanz/vscode-hie-server/pull/93 will add it for us, but its not merged yet

Reply:
I know. A better workaround for now would be to make a local batch/shell script to launch the wrapper with the lsp flag.

Reply:
Or even easier, change default executable to hie-wrapper.